code/game/turfs/simulated/floor/misc_floor.dm

```diff
@@ -210,6 +213,18 @@
 		animate(src, color = previouscolor, time = 8)
 		addtimer(CALLBACK(src, /atom/proc/update_atom_colour), 8)
 
+/turf/open/floor/clockwork/reebe
+	name = "cogplate"
+	desc = "Warm brass plating. You can feel it gently vibrating, as if machinery is on the other side."
+	icon_state = "reebe"
+	baseturf = /turf/open/floor/clockwork/reebe
+	uses_overlay = FALSE
+
+/turf/open/floor/clockwork/reebe/ex_act()
+	return
```

Comment:
Intentional; reebe floors aren't meant to be damageable, just everything on them